Resources/Help/analyses/ANCOVA.md

```diff
@@ -0,0 +1,227 @@
+- Assume equal variances: You can select this option when you assume that the variances of the levels of the independent variable are similar. This option is selected by default. 
+- Confidence Intervals: By selecting this option, confidence intervals for the estimated mean difference will be given. By default this is set to 95%. You can change this into the desired percentage. 
+
+#### Post Hoc Tests: 
+If there is a significant effect of the independent variable when is accounted for the covariate, you can follow up the analysis by performing a post hoc test to see which levels of a independent variable exactly differ. To perform a post hoc test drop the factor to perform the post hoc test on in the right column. Then you can select:    
+- Effect size: By selecting this option the effect size, referred as the magnitude of the observed effect, will be presented. The used measure for the effect size is Cohen's d. The effect size will only be presented for the post hoc type `Standard`. 
+- Confidence intervals: When this option is selected, the confidence interval for the mean difference is calculated. This is done for every post hoc method except for Dunn. By default this is set to 95% but you can adjust this into the desired percentage.  
+- Correction: To correct for multiple comparison testing and avoid Type I errors, you can select different correction methods. 
+    - Tukey: Compare all possible pairs of group means. This correction can be used when the groups of the independent variable have the same sample size and variance. It is a considered a common used correction and is selected by default. 
+    - Scheffe: Adjusting significance levels in a linear regression, to account for multiple comparisons. This method is considered to be very strict on Type I errors. 
+    - Bonferroni: This correction is considered conservative. The risk of Type I error is reduced, however the statistical power decreases as well. 
+    - Holm: This method is also called sequential Bonferroni, and considered less conservative than the Bonferroni method. 
+- Type: You can select different types of post hoc tests. 
+    -  Standard: Multiple t-tests are performed. All the corrections can be applied to this method. This option is selected by default. 
+    -  Games-Howell: This method can be used when you do not assume equal group/level variances. The Tukey correction is given for this method.  
+    -  Dunnett: When selecting this method, all the levels are compared to one specific level, for example to the control group. 
```

Comment:
Explain that we compare to the first level of the factor (for now it is not possible to manually specify which to compare to). In order to change the order of the levels, the level labels can be adjusted (give link to gif, https://jasp-stats.org/wp-content/uploads/2018/01/LabelEditing.gif).